pkg/awsutils/awsutils.go

```diff
@@ -1922,18 +1938,18 @@ func (cache *EC2InstanceMetadataCache) getENIsFromPaginatedDescribeNetworkInterf
 	return innerErr
 }
 
-// SetCNIUnmanagedENIs Set unmanaged ENI set
-func (cache *EC2InstanceMetadataCache) SetCNIUnmanagedENIs(eniID []string) error {
+// SetMultiCardENIs Set unmanaged ENI set
```

Comment:
nit: we should fix this comment to be something like: `SetMultiCardENIs creates a StringSet tracking ENIs not behind the default network card index`